### PR TITLE
Turn vsync into a core service.

### PR DESCRIPTION
**Pull request by cramforce**
Avoids loading it redundantly into extensions.

Part of #1055

### CODE REVIEW

File: src/amp-core-service.js [hunk 1]
```diff
@@ -28,4 +29,5 @@ export function installCoreServices(window) {
   installViewerService(window);
   installViewportService(window);
   installHistoryService(window);
+  installVsyncService(window);
```
Comment (dvoytenko) [inline src/amp-core-service.js]: Seems like vsync should be one of the first to be installed, or maybe right after viewer.

File: test/size.txt [hunk 2]
```diff
@@ -1,24 +1,24 @@
       max   |         min   |                                           gzip   |   file
       ---   |         ---   |                                            ---   |   ---
 295.34 kB   |    65.17 kB   |                                          cc.js
-461.34 kB   |   150.65 kB   |                                       41.35 kB   |   v0.js / amp.js
-126.19 kB   |    41.92 kB   |                                       15.04 kB   |   v0/amp-analytics-0.1.js
+462.41 kB   |   150.87 kB   |                                       41.37 kB   |   v0.js / amp.js
```
Comment (dvoytenko) [inline test/size.txt]: Really weird that it goes up by 1KB here. Isn't it?

**Comment by dvoytenko** [comment]
Thanks! A couple of questions, but otherwise LGTM.

**Comment by cramforce** [comment]
The increase in the core binary is because of one extra module. This could
be compiled away with a better compiler like closure or a custom pass to
browserify.

File: src/amp-core-service.js (hunk 1 above)
Comment (cramforce) [inline src/amp-core-service.js]: I have these install each other when needed, so that order here doesn't matter.

Comment (dvoytenko) [inline src/amp-core-service.js]: Understood. LGTM.

**Pull request merged**